jekyll/_cci2/runner-server.md

```diff
@@ -1,70 +1,24 @@
 ---
 layout: classic-docs
-title: CircleCI Runner in Server
-short-title: CircleCI Runner in Server
+title: Server launch-agent compatibility
+short-title: Server launch-agent compatbilitiy
 categories: [platforms]
-description: How to set up a Runner in Server
+description: Launch-agent compatibility table for server
 order: 31
 version:
 - Server v3.x
 ---
 
-NOTE: CircleCI Runner is not yet available on the current version of Server 3.x; however, it will be available for preview before the next release. Please [contact us](https://circleci.com/contact/) for more information.
+NOTE: CircleCI Runner is not yet available on the current version of Server
+3.x; however, it will be available for preview before the next release. Please
+[contact us](https://circleci.com/contact/) for more information.
```

Comment:
```suggestion
NOTE: CircleCI Runner is not yet available on the Server
3.0.x, however, it will be available for preview before the next release. Please
[contact us](https://circleci.com/contact/) for more information.
```